docs/articles/postComment.json

```diff
@@ -19,6 +19,10 @@
             "body": {
               "type": "string",
               "example": "just some comment"
+            },
+            "highlightedText": {
```

Comment:
Nice job guys. I like the thought process for implementing this. 
So I noticed you forgot to include the `highlightedText` field in the attributes section where we included comments on fetchArticles. Without this, we can't access this field on comments that have highlight text.

Reply:
We are only fetching the comments when we fetch single article. We don't need to get comments when fetching all articles.

Reply:
Cool. That totally makes sense.